app/ios/iosutils.h

```diff
@@ -39,11 +41,14 @@ class IosUtils: public QObject
   signals:
     void imageSelected( const QString &imagePath );
     void positionKitChanged();
+    void compassChanged();
 
   private:
 
     IOSImagePicker *mImagePicker = nullptr;
     QgsQuickPositionKit *mPositionKit = nullptr;
+    // TODO
+    PositionDirection *mCompass = nullptr;
```

Comment:
in the whole PR, mCompass should be new class "Compass". I assume this is still WIP?